timeseries/src/autogluon/timeseries/metrics/point.py

```diff
@@ -359,3 +360,54 @@ def __call__(
             seasonal_period=seasonal_period,
             **kwargs,
         )
+
+
+class CE(TimeSeriesScorer):
+    r"""Cumulative error.
+
+    This error measures the discrepancy between the cumulative sum of the forecast and the cumulative sum of the actual
+    values.
+
+    .. math::
+
+        \operatorname{CE} = \frac{1}{N} \frac{1}{H} \sum_{i=1}^{N} \sum_{t=T+1}^{T+H} \alpha1 \cdot \max(0, -d_{i, t}) + \alpha2 \cdot \max(0, d_{i, t})
+
+    where :math:`d_{i, t}` is the difference between the cumulative predicted value and the cumulative actual value
+
+    .. math::
+
+        d_{i, t} = \left(\sum_{t=T+1}^t f_{i, t}) - \left(\sum_{t=T+1}^t y_{i, t})
+
+    Parameters
+    ----------
+    alpha1 : float
+        Penalty assigned to underpredictions (when cumulative forecast is below the cumulative actual value).
+    alpha2 : float
+        Penalty assigned to overpredictions (when cumulative forecast is above the cumulative actual value).
+    """
+
+    def __init__(self, alpha1: float = 0.75, alpha2: float = 0.25):
```

Comment:
Let's have a single alpha which is equivalent to alpha1.  alpha2 would be replaced with 1 - alpha.

Reply:
Should we then maybe multiply the overall score by 2.0 so that in case alpha=0.5 we recover `MAE(np.cumsum(y_true, np.cumsum(y_pred))`?

Reply:
Yes, I like the idea.  That would make the score more intuitive.